smtpd/smtp_session.c

```diff
@@ -1833,27 +1833,28 @@ smtp_proceed_data(struct smtp_session *s, const char *args)
 static void
 smtp_proceed_quit(struct smtp_session *s, const char *args)
 {
-	smtp_reply(s, "221 %s: Bye",
+	smtp_reply(s, "221 %s Bye",
 	    esc_code(ESC_STATUS_OK, ESC_OTHER_STATUS));
 	smtp_enter_state(s, STATE_QUIT);
 }
 
 static void
 smtp_proceed_noop(struct smtp_session *s, const char *args)
 {
-	smtp_reply(s, "250 %s: Ok",
+	smtp_reply(s, "250 %s Ok",
 	    esc_code(ESC_STATUS_OK, ESC_OTHER_STATUS));
 }
 
 static void
 smtp_proceed_help(struct smtp_session *s, const char *args)
 {
-	smtp_reply(s, "214- This is " SMTPD_NAME);
-	smtp_reply(s, "214- To report bugs in the implementation, "
-	    "please contact bugs@openbsd.org");
-	smtp_reply(s, "214- with full details");
-	smtp_reply(s, "214 %s: End of HELP info",
-	    esc_code(ESC_STATUS_OK, ESC_OTHER_STATUS));
+	const char *esc_code = esc_code(ESC_STATUS_OK, ESC_OTHER_STATUS);
```

Comment:
Just thinking out loud, and on the basis it's not been compiled and tested... wouldn't there be a conflict between variable and function names being identical?
To remove any ambiguity, I suggest changing the local variable name to something like `code_esc`.